docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
     build: ./catalogs
     image: noaa-flood-catalogs
     environment:
-      - AWS_PROFILE=noaa-flood
+      - AWS_PROFILE=${AWS_PROFILE:-noaa}
```

Comment:
Updating this here for consistency with the profile we use in `docker-compose.ci.yml`.